mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/159 build exe (resolves #159)

Closes #159 
To test:

* follow the steps in the README
* should build the executable
* launch it and run a scenario
